src/postprocessing/Pass.ts

```diff
@@ -0,0 +1,63 @@
+import { OrthographicCamera, PlaneGeometry, Mesh, Material, Renderer, WebGLRenderer, WebGLRenderTarget } from 'three'
+
+class Pass {
+  // if set to true, the pass is processed by the composer
+  enabled = true
+
+  // if set to true, the pass indicates to swap read and write buffer after rendering
+  needsSwap = true
+
+  // if set to true, the pass clears its buffer before rendering
+  clear = false
+
+  // if set to true, the result of the pass is rendered to screen. This is set automatically by EffectComposer.
+  renderToScreen = false
+
+  setSize(
```

Comment:
these methods should be highlighted as `public` the same should be done for other methods unless they should be `private`

Reply:
If you feel strongly about this, I think we should enforce it explicitly through eslint. I don't think everyone will remember to do that, especially since methods are public by default. https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md

Reply:
I tried enabling this in a branch but got 238 errors in other TS files. I definitely don't want to look up the interfaces for all of those as a pre-req to merging this, if that's okay?

Reply:
👍🏼